src/renderer/js/page/file/view.jsx

```diff
@@ -62,6 +66,35 @@ class FilePage extends React.PureComponent {
       Object.values(player.mime).indexOf(contentType) !== -1 ||
       mediaType === "audio";
 
+    const channelClaimId =
+      value &&
+      value.publisherSignature &&
+      value.publisherSignature.certificateId;
+
+    const canSubscribe = !!channelName && !!channelClaimId;
```

Comment:
when would this happen?

Reply:
If someone posts an anonymous video. It might be fine with just the `!!channel` check. If there is no `channelName` there will be no `channelClaimId` (i think?)

Reply:
@kauffj any problems with only checking for `channelName`?

Reply:
FWIW, that is what I would do ^